queenfiona/SendITc3
# #161967625 Database configuration and project structure

#### What does this PR do?
Creates challenge 3 base branch develop, setups database configuration and project structure
#### Description of Task to be completed?
Setup database configuration and project structure
#### How should this be manually tested?
Clone this repository, cd into it, in terminal type tree to see the project structure
#### What are the relevant pivotal tracker stories?
#161967625
